### PR TITLE
Refs task #387243 Update payload

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets

- [#387243](https://dev.sun-asterisk.com/issues/387243)

## What's this PR do ?

- [x] Coding for update payload
- [x] Adding condition example for create payload

## Library
*(List package, library third party add new include version)*

- N/A

## Performance

- [ ] Resolved n + 1 query
- [ ] Run explain query already
- [ ] Time run rake task : 1000 ms

## Checklist

- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note purpose/reason, solution, scope of influence into ticket

## ENV note

```
```

## Notes
*(Other notes)*

- Form edit payload

image

- Condition example

image

### CODE REVIEW

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr></table>

File: routes/web.php [hunk 1]
```diff
@@ -39,5 +39,5 @@
     Route::resource('rooms', 'RoomController')->only([
         'index'
     ]);
-    Route::resource('webhooks.payloads', 'PayloadController')->except(['show', 'update']);
+    Route::resource('webhooks.payloads', 'PayloadController')->except(['show']);
```
Comment (ghost) [inline routes/web.php]: `->except('show')` xem dk ko chu

File: app/Http/Controllers/PayloadController.php [hunk 2]
```diff
@@ -98,9 +100,32 @@ public function edit(Webhook $webhook, Payload $payload)
      * @param  \App\Models\Payload  $payload
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, Payload $payload)
+    public function update(Request $request, Webhook $webhook, Payload $payload)
     {
-        //
+        $conditions = $request->only('conditions');
+        $data = $request->only('content');
+        $ids = $request->only('ids') ? $request->only('ids')['ids'] : [];
```
Comment (ghost) [inline app/Http/Controllers/PayloadController.php]: ids này có thể lấy từ conditions cũng dk, nếu ko muốn thì chú check ids = [] bên js cho nó dễ hơn

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr></table>

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr></table>

**Review by quanghung97 — APPROVED** [review]
ok

**Review by tamtt-1031 — APPROVED** [review]

**Review by nguyenthinhhn — APPROVED** [review]

**Comment by phucqd-1305** [comment]
OK

**Pull request merged**